_posts/2016-06-20-sustaining-hoodie.md

```diff
@@ -0,0 +1,91 @@
+
+Based on our experience, we run things differently with Hoodie. We know that technology is not the biggest challenge. Maintaining it, creating an inclusive community around it, and growing and sustaining it, while keeping it independent, are the biggest challenges.
+
+We think that a high adoption rate should not be Hoodie’s primary goal. If an independent Open Source project becomes too successful too fast, [the balance between users, contributors, and maintainers becomes unhealthy](https://medium.com/the-javascript-collection/healthy-open-source-967fa8be7951). A recipe for individual burnout and project failure.
+
+## Growing
+
+Our plan is to grow Hoodie in three stages:
+
+1. **The “Camp Release”:** focus on growing the team of maintainers and contributors (our current stage)
+2. **The “Village Release”:** focus on growing the ecosystem of plugins, tutorials, and supporting organisations
+3. **The Public Release:** focus on growing Hoodie’s user base
+
+So far the plan worked out great; we have a growing community of people with different talents and backgrounds. It took a lot of effort and we were only able to manage it because we were able to dedicate enough of our time to build a welcoming community. Now the question is: **how can we scale our efforts**?
+
+We think the answer is to put the right people in charge of it. The long-term goal is to have enough funds to pay people to work full-time on managing the Hoodie community. Our first milestone is a part-time position, working one day per week for Hoodie, 50 weeks per year.
+
+## Money in Open Source: Boon and Bane
+
+[Introducing money to a volunteering environment is tricky](https://www.aeaweb.org/articles?id=10.1257/jep.25.4.191).
+
+Once there is money available, it is tempting to pay people to work on things like code, or tests, or documentation. And I believe that it helps to increase contributions by paying for it, but in the long term paying contributors can become the bane of a project. Two examples:
+
+
+Contributors who get paid for their contribution once are less likely to contribute again.
+People will ask why one person gets paid while others don’t, for the same kind of work.
+
+I’m not concerned about traditional open source contributions. I think the real challenge is to scale the work we currently put into making Hoodie [the most welcoming community possible](http://hood.ie/blog/welcoming-communities.html).
+
+This kind of work is about _serving_ the community. Less like volunteers, more like employees for a non profit organisation. The work is labour intensive, it has to be done constantly and it is happening in the background, often times invisible to the rest of the community. Sometimes it’s just to be available, in case a moderation is needed in a discussion, or to enforce our code of conduct in a timely manner. And we have to hold people accountable for community work, which is much more practicable when paying for it.
```

Comment:
There could be an edit made here, either:

> in case a **moderator** is needed in a discussion

Or

> in case **moderation** is needed in a discussion